lib/hooks.h

```diff
@@ -20,6 +20,8 @@
 #include <jose/jws.h>
 #include <jose/jwe.h>
 
+#define MAX_COMPRESSION_SIZE (256*1024)
```

Comment:
his should be MAX_COMPRESSED_SIZE as it indicates the biggest compressed data you are willing to handle